SelfAndPeerEvaluations/partner_eval_quiz_creator.py

```diff
@@ -0,0 +1,161 @@
+import canvasapi
+import datetime
+import json
+from typing import Optional
+from utils import make_student_id_map
+
+
+class PartnerEvalQuizCreator:
+    def __init__(
+        self,
+        course: canvasapi.course.Course,
+        json_questions: dict,
```

Comment:
Type hinting for dict's can include their key and value types. Add this type def to line 7 and 8
 ```
JsonValue = Union[str, int, float, bool, list['JsonValue'], 'JsonDict']
JsonDict = dict[str, JsonValue]
```

```suggestion
        json_questions: JsonDict,
```